src/tile.cpp

```diff
@@ -496,28 +496,28 @@ ReturnValue Tile::queryAdd(int32_t, const Thing& thing, uint32_t, uint32_t flags
 			return RETURNVALUE_NOTPOSSIBLE;
 		}
 
-		if (const Monster* monster = creature->getMonster()) {
+		if (auto monster = creature->getMonster()) {
```

Comment:
Could this be `const auto`
```suggestion
		if (const auto monster = creature->getMonster()) {
```